hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/api/Query.kt

```diff
@@ -1,15 +1,17 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api
 
+import com.fasterxml.jackson.annotation.JsonBackReference
 import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.AttributeSearchException
 
 @Schema(description = "A query to search for prisoners by attributes")
 data class Query(
-  @Schema(description = "The type of join to use when combining the matchers and subQueries", example = "AND")
-  val joinType: JoinType,
+  @Schema(description = "The type of join to use when combining the matchers and subQueries", example = "AND", defaultValue = "AND")
+  val joinType: JoinType = JoinType.AND,
```

Comment:
Default JoinType to AND so it can be ommitted in the request. Particularly useful where there is a single query as join doesn't make sense to the client.